crsf.md

```diff
@@ -1121,6 +1122,28 @@ MSP frame over CRSF Payload packing:
 
 ## **0x80 Ardupilot reserved**
 
+## **0x81, 0x82 mLRS reserved**
+
+The 0x80 and 0x81 frames establish the communication of mLRS TX modules with mLRS Configuration Lua scripts running on e.g. EdgeTx and OpenTx remote controllers. They encapsulate mLRS 'mBridge' protocol messages in CRSF frames. The protocol enables the setting of parameters and other functionality, but especially also communicates meta data which are needed for mLRS' parameter model, providing various information to the user, version control, and more features specific to the mLRS link system.
+
+**0x80**: Communication from Lua script/remote controller to TX module. The frame format is:
+
+```mermaid
+flowchart LR
+    0xEE ~~~ id0[Frame Length] ~~~ 0x80 ~~~ 0x4F ~~~ 0x57 ~~~ id1[Command Byte] ~~~ Payload ~~~ CRC
```

Comment:
This looks like it follows the extended frame format and you use some undocumented addresses. Can you include those in the Device Addresses list or are these magic numbers?

Reply:
no. These two bytes 0x4f 0x57 are part of the mLRS mBridge protocol

Reply:
@Ilidur I could cahnge the first sentence to "The 0x80 and 0x81 boradcast frames establish..." if you think this could help a bit

Reply:
yes please. that will at least place it better in the spec. 
I would also ask that you describe those two special numbers. Either they are static magic numbers or they have a meaning and could be substituted with the meaning in the spec above.

Reply:
@Ilidur done.